### ISSUE TITLE
Bug in HTML output with 2 select controls

### ISSUE THREAD

**Issue by gitfoxi** [ioam/holoviews#1748]
My first day, so maybe user error but my HoloMap select control widgets works like I expect in the notebook:

![image](https://user-images.githubusercontent.com/584005/28549130-ffddd656-708c-11e7-9de7-c0ecbcf41534.png)

But when I export HTML, the options in one control have been copied into the other:

![image](https://user-images.githubusercontent.com/584005/28549166-2614b722-708d-11e7-905e-00b8668569e8.png)

See there was never an FF value in pattern.

The code that made this is a big notebook with much data, and quite a mess, so it may be work to make a testcase. But, for reference, the cell looks like:

```
%%output filename="Red-green-dashboard" fig="html"
%%opts Scatter [tools=['hover'] width=600 height=600 toolbar='right' color_index=2] (cmap='RdYlGn', size=10, line_color='black', alpha=0.2)

# Try to make some red-green shmoos

curves = hv.HoloMap({id: 
    hv.Scatter(grp, kdims=['x'], vdims=['y', 'pf', 'part_num'])
                    for id, grp in df.groupby(['pattern', 'setpoint', 'split'])},
                    key_dimensions=['pattern', 'setpoint', 'split']
                   )
curves
```

**Comment by jlstevens** [comment]
Very odd! Tagging this as a bug...

**Comment by tv3141** [comment]
Similar issue #1871 has been fixed.

**Comment by jlstevens** [comment]
Thanks for pointing that out @tv3141 !

@gitfoxi Could you checkout the latest master and see if this is still an issue? Alternative, you can try 1.8.4 which has just been released on conda (``conda install -c ioam holoviews``) and pip (``pip install holoviews).

**Comment by philippjfr** [comment]
As @tv3141 points out this bug was fixed in #1871. Please reopen if you encounter any issues.
